crawl-ref/source/dat/des/altar/overflow.des

```diff
@@ -2354,6 +2364,44 @@ xxxxxx.xxxxxx
      x@x
 ENDMAP
 
+NAME:  pdpol_wu_jian_five_deadly_venoms
+DEPTH: D:2-, Lair
```

Comment:
```suggestion
DEPTH: D:2-9
```

Overflow vaults are restricted to this specific depth, they get special handling in the level builder.